rpcs3/rpcs3qt/gui_settings.h

```diff
@@ -130,6 +130,8 @@ namespace gui
 	const gui_save fd_cg_disasm    = gui_save(main_window, "lastExplorePathCGD",  "");
 	const gui_save fd_log_viewer   = gui_save(main_window, "lastExplorePathLOG",  "");
 	const gui_save fd_ext_mself    = gui_save(main_window, "lastExplorePathExMSELF",  "");
+	const gui_save fd_ext_pup      = gui_save(main_window, "lastExplorePathExPUP",  "");
```

Comment:
why is this an extra entry?
I'd assume people would use the same dir as fd_install_pup